src/SonarScanner.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -168,6 +169,15 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="WriteSolutionTelemetry" Condition=" $(SonarQubeTempPath) != ''">
```

Comment:
This Target is not executed. It needs to be called either by `<CallTarget Targets="WriteSolutionTelemetry" />` from another target or by specifying a `BeforeTargets` or `AfterTargets` attribute.